draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1181,7 +1181,7 @@ Repeat(10,
 Update(M0, M1)
 ~~~
 
-The AEGIS-128X `Update` function is similar to the AEGIS-128L `Update` function, but absorbs `R` (`2 * 128 * D`) bits at once. `M0` and `M1` are `128 * D` bits instead of 128 bits but are split into 128-bit blocks, each of them updating a different AEGIS-128L state.
+The AEGIS-128X `Update` function is similar to the AEGIS-128L `Update` function, but absorbs `R` (= `256 * D`) bits at once. `M0` and `M1` are `128 * D` bits instead of 128 bits but are split into 128-bit blocks, each of them updating a different AEGIS-128L state.
```

Comment:
Do you prefer this with or without the equals sign?

Reply:
With the context, I don't think `R (256 * D)` can be interpreted as `R * (256 * D)`, but without the context, I understand that it could, so adding a separator doesn't hurt.

We could write something in English ("equaling"?) but that looks weird.

Reply:
Fair point, happy to leave it as is.